cmd/oc/oc.go

```diff
@@ -7,7 +7,7 @@ import (
 	"runtime"
 	"time"
 
-	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
```

Comment:
this file should be touched.

Reply:
I agree this bit should not be touched, but we still need to install the configapi into kubectl's scheme, no?